libs/net/models/Areas/Editor/Models/AVOverview/AVOverviewInstanceModel.cs

```diff
@@ -85,6 +85,8 @@ public AVOverviewInstanceModel(Entities.AVOverviewTemplate entity, DateTime publ
             Response = model.Response,
             Version = model.Version ?? 0
         };
+        if (!string.IsNullOrEmpty(model.UpdatedBy)) entity.UpdatedBy = model.UpdatedBy;
```

Comment:
Why are you doing this?  We set these values automatically already.  Refer to the `TNOContext.SaveChanges()` code.

Reply:
It always shows missing this info for some reason when in local debugging mode, however, the change is unnecessary and now removed, thanks